docs/bookstore/page-0.5/pg0.5-env-setup.md

```diff
@@ -0,0 +1,229 @@
+
+# Environment Setup
+
+![Image](images/image20.png)
+
+In this page, we will be discussing how to set up your environment, and make sure to run up the UI front end and the Book Review Service’s node server.
+
+## **What does the final deliverable for this section look like?**
+
+- You have a running kubernetes (k8s) cluster on your local laptop, with Knative Installed.
+- You have your front end application deployed as kubernetes deployment with port-forwarding to localhost:3000
+- You have your Node.js application deployed as kubernetes deployment with port-forwarding to localhost:8080
+
+We will be fulfilling each requirement with the order above.
+
+## **File Structure**
+
+![Image](images/image21.png)
+
+The code for the sample app is in `docs/code-samples/eventing/bookstore-sample-app`
+
+Under bookstore-sample-app folder, there are 2 folders:
+
+**/solution**: this folder contains all the yaml files, and the code you needed. Check it when you get stuck.
+
+**/start**: this folder contains the necessary files for you to get started. Write all the configuration files yourself following the tutorial!
```

Comment:
```suggestion
* **/solution**: this folder contains all the yaml files, and the code you needed. Check it when you get stuck.

* **/start**: this folder contains the necessary files for you to get started. Write all the configuration files yourself following the tutorial!
```
Maybe use a list for better structure.

And not sure what you mean with `Write all the configuration files yourself following the tutorial!`. You mean something like "Save here all the configuration files yourself following the tutorial!"?